frontend/compositions/officer-view/officer-affiliations/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { OfficerRecordType } from "../../../models/officer"
+import styles from "./officer-affiliations.module.css"
+
+export default function OfficerAffiliations(officer: OfficerRecordType) {
+  const { affiliations } = officer
+
+  let affiliationsString = ""
```

Comment:
lets get rid of the let, not a van of it since it is mutable. We can also take advantage of JS built in functions I would change line 7 thorugh line 10 to 
`const affiliationString = affiliations.join(', ')` 
Much cleaner and declarative.